README.md

```diff
@@ -60,6 +60,50 @@ Images to be included in the docs are kept in `source/images`
 
 In order to configure some aspects of layout, like the header, edit `config/tech-docs.yml`.
 
+## Making changes to diagrams
+
+To make changes to the diagrams, edit the `drawio` files in `source/images/originals` using the drawio app.
+
+
+### Workflow
+
+You need to:
+
+- create and modify diagrams using [draw.io](https://about.draw.io/)
+- store draw.io master files in `source/images/originals`
+- publish diagrams as [Scalable Vector Graphics (SVG)](https://www.w3.org/Graphics/SVG/)
+- store svg image files in the `source/images` folder
+
+### Prerequisites
+
+Install the draw.io desktop application for interactive editing
+
+```
+brew cask install drawio
+```
+
+Alias the binary so that your can just type `draw.io` at the command line
+
+```
+alias=draw.io='/Applications/draw.io.app/Contents/MacOS/draw.io'
+```
+
+### Useful commands
+
+Update a diagram
+
+```
+draw.io source/images/originals/top-level-technical-diagram.drawio
+```
+
+Generate an SVG versions of the diagrams
+
+```
+draw.io -x -o source/images/top-level-technical-diagram.svg source/images/originals/top-level-technical-diagram.drawio
+
+draw.io -x -o source/images/technical-flow-diagram.svg source/images/originals/technical-flow-diagram.drawio
+```
+
```

Comment:
````suggestion
## Making changes to a diagram

To make changes to a diagram, you'll need to edit the `drawio` files in `source/images/originals` using the draw.io desktop app.

### Install draw.io desktop app

1. Install the draw.io desktop app using Homebrew with the following command: 

```
brew cask install drawio
```

2. Alias the binary so that you can just type `draw.io` at the command line. Use the following command to do this:

```
alias=draw.io='/Applications/draw.io.app/Contents/MacOS/draw.io'
```

### Edit and publish the diagram

Follow these steps:
1. Create and modify the diagram using [draw.io](https://about.draw.io/).
1. Store the draw.io original file in `source/images/originals`.
1. Publish the diagram as a [scalable vector graphic (SVG)](https://www.w3.org/Graphics/SVG/).
1. Store the SVG in the `source/images` folder.

### Useful commands

Update a diagram:

```
draw.io source/images/originals/top-level-technical-diagram.drawio
```

Generate SVG versions of the diagrams:

```
draw.io -x -o source/images/top-level-technical-diagram.svg source/images/originals/top-level-technical-diagram.drawio

draw.io -x -o source/images/technical-flow-diagram.svg source/images/originals/technical-flow-diagram.drawio
```
````

Reply:
Have suggested we could leave the README update out of this PR to get the refresh token changes in ASAP.

Reply:
Hi @sabrinaharris I have pulled the README update and will make another PR to tackle this after the content has shipped. Its only me making the diagram changes at the moment so not a problem. Longer term it needs to be documented

Reply:
Thanks very much @pauldougan!

Reply:
Created [issue for README changes ](https://govukverify.atlassian.net/jira/software/c/projects/AP/boards/452?selectedIssue=AP-166) 👍